api/novelist.py

```diff
@@ -549,7 +554,7 @@ def create_item_object(self, object, currentIdentifier, existingItem):
             # If we encounter a new ISBN, we take whatever author value is
             # initially given to us.
             title = object[3]
-            mediaType = Edition.medium_to_book_format_type_values[object[4]]
+            mediaType = self.medium_to_book_format_type_values[object[4]]
```

Comment:
This will work, but just to avoid crashes, do self.medium_to_book_format_type_values.get(object[4], ""). That will handle anything unusual in the database.